test/unit/uk/gov/hmrc/thirdpartyapplication/services/SubscriptionServiceSpec.scala

```diff
@@ -60,29 +60,20 @@ class SubscriptionServiceSpec extends AsyncHmrcSpec with BeforeAndAfterAll with
     )
 
     val underTest = new SubscriptionService(
-      mockApplicationRepository, mockSubscriptionRepository, ApiDefinitionConnectorMock.aMock, AuditServiceMock.aMock, mockApiGatewayStore)
+      mockApplicationRepository, mockSubscriptionRepository, ApiDefinitionConnectorMock.aMock, mockAuditService, mockApiGatewayStore)
```

Comment:
Pattern would be to add the calls to the AuditServiceMock rather than rebuilding it all.  I appreciate it's maybe not used elsewhere yet (not every location was moved over to shared mocks)